server/controllers/recall/v2RecallFlow/selectCourtCaseControllerV2.ts

```diff
@@ -219,7 +219,7 @@ export default class SelectCourtCaseControllerV2 extends BaseController {
           const enhancedCases = res.locals.recallableCourtCases as EnhancedRecallableCourtCase[]
 
           reviewableCases = enhancedCases
-            .filter(c => c.status !== 'DRAFT' && c.isSentenced)
```

Comment:
Can we figure out what the original intention of filtering out DRAFT court cases was?? (i.e. what did the analyst intend)